src/dearpygui_commands.h

```diff
@@ -2631,6 +2786,11 @@ configure_app(PyObject* self, PyObject* args, PyObject* kwargs)
 
 	if (PyObject* item = PyDict_GetItemString(kwargs, "keyboard_navigation")) GContext->IO.kbdNavigation = ToBool(item);
 
+	ImGuiStyle& style = ImGui::GetStyle();
+	if (PyObject* item = PyDict_GetItemString(kwargs, "anti_aliased_lines")) style.AntiAliasedLines = ToBool(item);
```

Comment:
`anti_aliased_lines`, `anti_aliased_lines_use_tex`, `anti_aliased_fill`: For some reason, these arguments are accepted by `configure_app` but are missing from `get_app_configuration`. We need to add them there.